lib/maze/servlets/servlet.rb

```diff
@@ -115,9 +115,9 @@ def do_POST(request, response)
           $logger.warn msg
         end
       rescue StandardError => e
-        Bugsnag.notify e
         if Maze.config.captured_invalid_requests.include? @request_type
-          $logger.error "Invalid request: #{e.message}"
+          Bugsnag.notify e
+          $logger.error "Invalid #{@request_type} request: #{e.message}"
```

Comment:
I don't quite follow why this is an error and if it's not in the `captured_invalid_requests` list then it's a warning, but presumably you're intentionally not bugsnagging the latter?

My consistency alarm is sounding though as in the rescue block above you are bugsnagging for the warning case too (L105).

Reply:
You're right, the error handlers should be consistent with each other - corrected.

There are some circumstance where we don't care if a request we receive is invalid.  For example, a request made to `/command` from a test fixture polling away, or in a test scenario where we don't care about session requests but can't turn them off completely for some reason.  We still log it as a warning though to avoid it being completely invisible to the test scenario - just in case there is a significance that we didn't realise.